packages/upgrade/source/guide-generators/v5/retheme/index.js

```diff
@@ -5,14 +5,22 @@ const cwd = 'v5/retheme';
 async function generate() {
   const data = (await import(`../../../versions/v5/index.js`)).default.nextjs;
 
+  const appearancePropBreakingChanges = {
+    title: 'Breaking Changes to appearance variables',
+    content: `\nSeveral appearance variables have been removed or renamed. If you were using these variables in your application, you will need to update your code to use the new variables.
+
+    - The \`fontSmoothing\` variable has been removed.
+    - The \`shadowShimmer\` variable has been removed.
+    - The \`colorAlphaShade\` variable has been renamed to \`colorNeutral\`.\n`,
```

Comment:
Is there no realistic way we can regex scan for these? Assuming so but just want to double check

Reply:
@jescalan Oh, I think we can! I will add a commit.

Reply:
If so, can be written up as a markdown file inside `versions/v5/common` folder and referenced from there using `singleItem` 😁